frontend/src/concepts/areas/const.ts

```diff
@@ -73,6 +74,14 @@ export const SupportedAreasStateMap: SupportedAreasState = {
   [SupportedArea.DS_PROJECTS_VIEW]: {
     featureFlags: ['disableProjects'],
   },
+  [SupportedArea.DS_PROJECT_SCOPED]: {
+    featureFlags: ['disableProjectScoped'],
+    reliantAreas: [
```

Comment:
@mturley Does this section look good?

Reply:
Hmm, this means the feature will only work if all 3 reliantAreas are enabled - do we want this to work for workbenches and hardware profiles if model serving is disabled for example? If so it might be fine to just not have any reliantAreas here

Reply:
Yes, we need it to work with whatever component is available. Removing the `reliantAreas`

Reply:
On second thoughts, `reliantAreas` only needs one to be true as per https://github.com/opendatahub-io/odh-dashboard/blob/92357499cc533936aa7d9e24a77f2dbde333f28e/frontend/src/concepts/areas/types.ts#L16

Reply:
I think we don't need a ProjectScoped if all three are turned OFF, although workbenches will always be turned ON by default coz I guess it doesn't have a feature flag.

Reply:
Also, based on manual test for https://github.com/opendatahub-io/odh-dashboard/pull/3871, it was working for the other reliantAreas if one was absent.

Reply:
Ok interesting, I wasn't aware of reliantAreas working that way.... hmm, I wonder if we've made assumptions that all reliant areas are required in other places. Like SupportedArea.FINE_TUNING should really require all of its reliantAreas.... something to come back to...